graylog2-web-interface/src/components/sidecars/common/ColorLabel.tsx

```diff
@@ -46,16 +47,21 @@ const ColorLabelWrap = styled.span(({ size, theme }: ColorLabelWrapProps) => {
   `;
 });
 
-const ColorLabel = ({ color, size, text, theme }: ColorLabelProps) => {
+const ColorLabel = ({ color, size, text, theme, className }: ColorLabelProps) => {
   const borderColor = theme.utils.colorLevel(color, 5);
   const textColor = theme.utils.contrastingColor(color);
 
   return (
-    <ColorLabelWrap size={size}>
+    <ColorLabelWrap size={size} className={className}>
```

Comment:
While this works, the `className` prop already exists on react component. I suggest adding a `className` as below
```suggestion
    <ColorLabelWrap size={size} className="color-label-wrapper">
```
Then in the `CollectorAdministration.tsx` create a Styled component for `ColorLabel` and use the CSS class we defined above:
```
const StyledColorLabel = styled(ColorLabel)`
  .color-label-wrapper {
    display: flex;
  }
`;
```
This way we can remove the passed `className` prop.